### DISCUSSION TITLE
Problems with git hub education

### DISCUSSION THREAD

**Discussion by neiro2222** [community/community#138679] [GitHub Education]
### Hi everyone,

I applied 7 days ago but you didn"t send anything. Please help me

**Comment by pg56714** [comment]
If your application is approved, you will receive a confirmation email. Applications are generally processed within a few days, but during peak times, such as the start of a new semester, it may take longer.

If I am correct, please mark it as the answer. Thank you.

**Comment by davevad93** [comment]
Hi @neiro2222 ,

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @neiro2222 was approved and your benefits are now available.` 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
